src/Entity/ItemizedPaymentInformation.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Ups\Entity;
+
+use LogicException;
+
+class ItemizedPaymentInformation
+{
+
+    /**
+     * @var splitDutyVATIndicator
+     */
+    private $splitDutyVATIndicator;
+    private $transportationShipmentCharge = null;
+    private $dutiesAndTaxesShipmentCharge = null;
+
+    public function __construct($attributes = null)
```

Comment:
Constructor doesn't do anything, could you make the behavior similar to other entities?